e2e/testdata/fn-render/missing-fnconfig/.expected/config.yaml

```diff
@@ -18,6 +18,3 @@ stdErr: |
   Field: `pipeline.mutators[1].configPath`
   Value: "labelconfig1.yaml"
   Reason: functionConfig must exist in the current package
-stdOut: |
```

Comment:
Why remove this? Although this stdout check looks not necessary.

Reply:
This test case no longer has any stdout because the error occurs during validation now.